packages/block-editor/src/components/inserter-list-item/README.md

```diff
@@ -0,0 +1,59 @@
+# Inserter List Item
+
+The `InserterListItem` component allows you to display a group of elements for each of the blocks that are present in the editor's block type list interfaces.
+
+These groups are made up of a block icon and its name.
+
+In practice, a click on these groups allows you to insert the corresponding block in the editor.
+
+![Paragraph block inserter list item](https://make.wordpress.org/core/files/2020/09/paragraph-inserter-list-item.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Renders a block inserter list item.
+
+```jsx
+import { InserterListItem } from '@wordpress/block-editor';
+
+const MyInserterListItem = () => (
+	<InserterListItem />
```

Comment:
@ntsekouras maybe you can help me on this ... I'm having a hard time to identify what are the required props for this component.

Reply:
Hey @JustinyAhin! The `InserterListItem` component it's actually a single element that shows a block in quick appender and the main appender list. It's only used for now in these places with a single entry point here: https://github.com/WordPress/gutenberg/blob/9ced21b344695052db1b3fd10a865b205e5c418e/packages/block-editor/src/components/block-types-list/index.js#L52

You can see from the code above, the usage of many needed props in GB core.

I'm not really sure if the `composite` is required and if is tightly coupled with Reakit parent there.. (```import { Composite, useCompositeState } from 'reakit';```

(https://reakit.io/docs/composite/). @youknowriad any input here, since I saw you made that addition for better keyboard navigation.